### ISSUE TITLE
In 5.0.1 does not install bin/elastix and bin/transformix

### ISSUE THREAD

**Issue by slitvinov** [SuperElastix/elastix#425]
I updated from 5.0.0 to 5.0.1 and noticed that `bin/elastix` and `bin/transformix` are note being installed any more.

I do
```
cmake  . -DCMAKE_INSTALL_PREFIX=$HOME/.local
make
make install
```
Lots of files are copied to `$HOME/.local/include` and `$HOME/.local/lib` but nothing goes to `$HOME/.local/bin`.

**Comment by N-Dekker** [comment]
@slitvinov Thanks for reporting this issue, Sergey. Can you please check, as it seems to be a duplicate of issue #305 "Elastix 5.0.1 binaries (elastix and transformix) not installed by default cmake build", which was fixed with commit ed4f1b1a07036a40bbf2df716595a1ca12f9c166 "BUG: Add elastix_exe and transformix_exe to CMake install"  (6 October 2020)

**Comment by slitvinov** [comment]
@N-Dekker It is a duplicate. Thank you.
